src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -65,4 +65,23 @@ public static boolean isNonZeroUnsignedInteger(String s) {
             return false;
         }
     }
+
+    /**
+     * Returns true if {@code s} represents an unsigned integer inclusive of zero
+     * e.g. 0, 1, 2, 3, ..., {@code Integer.MAX_VALUE} <br>
+     * Will return false for any other non-null string input
+     * e.g. empty string, "-1", "+1", and " 2 " (untrimmed), "3 0" (contains whitespace), "1 a" (contains letters)
+     * @throws NullPointerException if {@code s} is null.
+     */
+    public static boolean isUnsignedIntegerIncludingZero(String s) {
```

Comment:
I think just being `isUnsignedInteger` is good enough. 
Unsigned value by definition already means no negative, so implicitly means including 0.

```suggestion
    public static boolean isUnsignedInteger(String s) {
```